src/studio/src/designer/frontend/.eslintrc.json

```diff
@@ -49,6 +49,12 @@
     ],
     "jsx-a11y/no-autofocus": ["off"],
     "@typescript-eslint/no-explicit-any": ["off"],
-    "import/no-unresolved": ["off"]
+    "import/no-unresolved": ["off"],
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "ignoreRestSiblings": true
+      }
+    ]
```

Comment:
Are we sure we want to introduce this rule? 🙂 Just thinking it could cause clutter to be left behind...

Reply:
Hmm you're probably right. We use restSiblings actively some places in the schemaEditor, but I can add a file-wide exception to the rule in the relevant places instead 🙂

Reply:
Not sure whats best here really. I equally dislike eslint comments in files, because they tend to get copied as well when copying code - so they easily spread to places they dont belong 😅. If we want to use this feature, perhaps its actually better to disable it altogether.

Reply:
Hehe yep 😅 Maybe we could add a separate eslint config for the schemaEditor, so we don't introduce this rule in places where it's not relevant at the moment?

Reply:
Sounds like a good plan 😁